src/lava/proc/lif/models.py

```diff
@@ -320,3 +320,110 @@ def reset_voltage(self, spike_vector: np.ndarray):
         """Reset voltage of all spiking neurons to 0.
         """
         self.v[spike_vector != 0] = 0  # Reset voltage to 0 wherever we spiked
+
+
+@implements(proc=LIFReset, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('floating_pt')
+class PyLifResetModelFloat(AbstractPyLifModelFloat):
+    """Implementation of Leaky-Integrate-and-Fire neural process with reset
```

Comment:
This description isn't very informative, but I guess the proc might have a better one?

Reply:
Yeah the process already has all the necessary details.